Library/Equipment/Low Tech.eqp

```diff
@@ -13030,7 +13030,7 @@
 		<melee_weapon>
 			<damage>thr imp</damage>
 			<strength>7</strength>
-			<usage>Thrust</usage>
+			<usage>Thrust - Shortsword</usage>
```

Comment:
Maybe I just don't grasp what this is trying to do, as I see more below where you are treating the weapon differently based on knife/shortsword, which doesn't make any sense to me.